bot/tests/sending_videos/test_select_clip.py

```diff
@@ -36,11 +36,3 @@ async def test_select_invalid_segment_number(self):
 
         response = await self.send_command('/wybierz 999')
         self.assert_response_contains(response, [get_invalid_segment_number_message()])
-
-    # @pytest.mark.asyncio
-    # async def test_select_clip_duration_exceeds_limit(self):
-    #     search_response = await self.send_command('/szukaj długi_segment')
-    #     await self.assert_message_hash_matches(search_response, expected_key="search_long_segment_results.message")
-    #
-    #     response = await self.send_command('/wybierz 2')
-    #     self.assert_response_contains(response, [msg.get_limit_exceeded_clip_duration_message()])
```

Comment:
nie potrafi wyjść poza limit czy co? XD

Reply:
No w zasadzie taka sytuacja nie występuje przez to jak whispar działa. Bo on sobie dzieli materiał na segmenty i chyba bez szans żeby podzielił na fragmenty po 3min xD. Nie wiem jakie zdania by musiały być żeby to wypluło taki segment xD. Co za tym idzie nie mam za bardzo fragmentu na którym bym mógł to testować.